text/0000-basic-template-helpers.md

```diff
@@ -0,0 +1,86 @@
+- Start Date: 2018-10-13
+- RFC PR: (leave this empty)
+- Ember Issue: (leave this empty)
+
+# (RFC title goes here)
+
+## Summary
+
+Add new built-in template helpers to perform basic operations in templates, identical to the
+helpers in [ember-truth-helpers](https://github.com/jmurphyau/ember-truth-helpers)
+
+## Motivation
+
+It is a very common necessity to almost every Ember app to perform certain operations like compare
+by equality, negate a value, or perform boolean operations, and often the most convenient place to
+do it is in the templates.
+Because of that, `ember-truth-helpers` is probably the single most installed addon either directly
+or transitively in the entire ecosystem.
+
+The fact that this addon is so popular is very telling and Ember should consider to move into the
+core of the templating engine at least the most popular helpers.
+
+One of the shocking moments developers that are familiar with React, Vue or Angular have when trying Ember
+is that they cannot, at least out of the box, perform the most basic comparisons and boolean operations
+they are so used to in JSX or Vue/Angular templates.
+
+Furthermore, if the [RFC #367](https://github.com/emberjs/rfcs/pull/367) eventually gets merged, these
+helpers are the perfect candidates to create adoption friction in the community because of how
+pervasive its usage it in so many templates.
+
+Albeit less important, implementing these helpers in the core _could_ enable clever trickery in the
+Glimmer VM, like evaluating them at compile time when the arguments are constant.
+
+## Detailed design
+
+The process consists on deciding what helpers from `ember-truth-helpers` we consider the most important
+and move them into Ember.js itself or even the Glimmer VM, in a fully backwards compatible way.
+
+I propose to move to core at least:
+
+- `eq`
```

Comment:
will that be `==` or `===` semantics?

Reply:
If we want to maintain backwards compatibility it has to be `===`. And even if compatibility wasn't a concern I'd still prefer triple equal